tests/php/src/Unit/GoogleFontsTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Tests for GoogleFonts class.
+ *
+ * @package AMP
+ */
+
+use AmpProject\AmpWP\Admin\GoogleFonts;
+use AmpProject\AmpWP\Infrastructure\Delayed;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+
+/**
+ * Tests for GoogleFonts class.
+ *
+ * @group google-fonts
+ *
```

Comment:
Why add a group if there's only 1 test case?

Sidenote: If the intent was testing this only you could filter the tests to run with:

```suggestion
```